src/main/java/dev/blaauwendraad/masker/json/path/package-info.java

```diff
@@ -0,0 +1,4 @@
+@ParametersAreNonnullByDefault
+package dev.blaauwendraad.masker.json.path;
+
+import javax.annotation.ParametersAreNonnullByDefault;
```

Comment:
Nice! I forgot you can do it like that

Reply:
It is from your commit that I rebased onto by the way :D

Reply:
That commit was really long time ago 😅